Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.js

```diff
@@ -159,6 +162,37 @@ WI.ConsoleMessageView = class ConsoleMessageView extends WI.Object
         this._repeatCountElement.textContent = Number.abbreviate(count);
     }
 
+    _renderTimestamp()
+    {
+        if (!this._timestamp) {
+            if (this._timestampElement) {
+                this._timestampElement.remove();
+                this._timestampElement = null;
+            }
+            return;
+        }
+
+        if (!this._timestampElement) {
+            this._timestampElement = document.createElement("span");
+            this._timestampElement.classList.add("timestamp");
+            this._element.insertBefore(this._timestampElement, this._element.firstChild);
+        }
+
+        let date = new Date(this._timestamp * 1000);
```

Comment:
the timestamp never changes, right?  if so, can we just early `return` if we've already created the `this._timestampElement`?  i.e. we shouldn't have to (re)generate a string for the timestamp if we've already done so